### DISCUSSION TITLE
How to change the state of icon in the taskbar?

### DISCUSSION THREAD

**Discussion by kermanx** [microsoft/terminal#12590] [Q&A] [answered]
When I run `winget`, I see 
![image](https://user-images.githubusercontent.com/63178754/155869160-b5ceafc9-c419-4121-92ab-2454ec840601.png)
in the taskbar.

However, I can't find any documention about it...

Waiting for your help..🙏

**Comment by kermanx** [comment]
And sometimes there can be a progress here.

**Reply by hello-smile6** [reply]
Is Windows maybe taking a progress indicator from a subprocess and showing it in place of the parent process's nonexistent indicator?

**Comment by zadjii-msft** [comment, marked as answer]
Best documentation I can offer at the moment is #6700. Apps can emit a special sequence to the Terminal to tell it "hey, please display a progress par with this much progress on the taskbar". The Terminal will also display that progress as a ring in the tab icon for that tab.

**Reply by zadjii-msft** [reply]
I've filed a new docs issue over at https://github.com/MicrosoftDocs/terminal/issues/514 to track adding this to the docs. For now, #6700 is gonna be your best bet, till the docs are updated.
